intelmq/lib/utils.py

```diff
@@ -876,6 +876,15 @@ def list_all_bots() -> dict:
     return bots
 
 
+def get_runtime() -> dict:
+    return load_configuration(RUNTIME_CONF_FILE)
```

Comment:
shouldn't global be removed here?

Reply:
I would not say so. The `get_runtime` and `set_runtime` functions read and write the `runtime.yaml` as a whole

Reply:
yeah, we can make a `get_effective_runtime(bot_id)` or so in the future, if we need it.

Reply:
If you're saying that the term `runtime` only refers to the individual configuration of the bots but not to the global settings which are now also part of `runtime.conf`, then it might make sense to rename the configuration file altogether (`intelmq.yaml`) and add one key `runtime` and one key `global`. That way we could use functions with meaningful names.